sdk/python/kfp/dsl/__init__.py

```diff
@@ -17,4 +17,5 @@
 from ._pipeline import Pipeline, pipeline, get_pipeline_conf
 from ._container_op import ContainerOp
 from ._ops_group import OpsGroup, ExitHandler, Condition
-from ._python_component import python_component
\ No newline at end of file
+from ._component import python_component
```

Comment:
nit: do you need to expose component annotation here?

Reply:
I wanted to expose the component decorator when fully tested. WDYT?